dmsite/venv/src/Datamaster.js

```diff
@@ -1,53 +1,27 @@
 import React, { Component } from 'react';
+import { Layout } from 'antd';
```

Comment:
Mind giving a brief description of what antd library is? or linking it

Reply:
https://ant.design/